datamade/how-to
# Migrate deploy-a-site docs to this repo

## Overview

As part of our migration to Heroku, migrate all active docs from the https://github.com/datamade/deploy-a-site/ repo to this repo. This includes:

- GPG
- Logging (Slack and Sentry)
- Static site deployments to Netlify
- Django file uploads

Handles #61.

### Notes

Once this PR has been pulled in, we can officially deprecate `deploy-a-site` in https://github.com/datamade/deploy-a-site/pull/82.

## Testing Instructions

* View the rendered markdown and confirm it looks good
* Test links and make sure they resolve correctly
